demo/run-tmux.py

```diff
@@ -0,0 +1,188 @@
+''
+# The above quotation is required because we import this
+# module as a nix `String`.
+# Please don't use triple quotes inside this `Python` module.
```

Comment:
This sounds fishy to me. Is this the canonical way of packaging a python script via nix?

Reply:
You are absolutely right. I was able to turn drop this bull**** and use just `builtin.readFile ./deme/run-tmux.py` - I'm to sure why I assumed that I can't do this :-)